Oyebola-O/my-recipe
# Update signup page

#### What does this PR do?
Updates the look of the of the Signup page

### Description of Task to be completed?
The css file was modified to style the form so that it stands out on the page

### How should this be manually tested?
After cloning the repo, cd into the auth folder. Open the  signup.html in your browser to see the changes

### What are the relevant pivotal tracker stories
Feature#1
